aquarius/events/processors.py

```diff
@@ -81,30 +84,40 @@ def add_aqua_data(self, record):
         block_info = self._web3.eth.get_block(self.event.blockNumber)
         block_time = datetime.fromtimestamp(block_info["timestamp"]).isoformat()
 
-        record["event"] = {
+        record[AquariusCustomDDOFields.EVENT.value] = {
```

Comment:
I don't generally like Enums. IntEnums are fine, since you can access everything with Class.ATTR, but I'd rather have the plain text instead of Class.ATTR.value. My reasoning for this is the fact that we do not reuse this much in other classes, and we are literally typing "value" to get a key, which is very confusing.

If you want to keep constants, II would suggest a simple class constant approach i.e. making it Class.ATTR instead of Class.ATTR.value.

For example, you can havee
class AquariusCustomDDOFields:
    EVENT
   (etc.)
   
   
   you can add a static function on AquariusCustomDDOFields to get the full list. e.g. get_all_keys()